README.md

```diff
@@ -25,7 +25,7 @@
 # Introduction
 
 `vicar` is a Python module that supports reading and writing of JPL's VICAR file format. It supports the definition of the VICAR file format as found here:
-[https://www-mipl.jpl.nasa.gov/external/VICAR_file_fmt.pdf](https://www-mipl.jpl.nasa.gov/external/VICAR_file_fmt.pdf)
+[https://pds-rings.seti.org/help](https://pds-rings.seti.org/help)
```

Comment:
This needs to be the full filename: https://pds-rings.seti.org/help/VICAR_file_fmt.pdf

Reply:
Fixed